index.bs

```diff
@@ -8618,9 +8611,9 @@ appears on an [=interface=]
 that is not defined to [=interface/inherit=]
 from another, it indicates that the internal \[[Prototype]]
 property of its [=interface prototype object=]
-will be the intrinsic object [=%ArrayPrototype%=] rather than
-[=%ObjectPrototype%=].  This allows
-Array methods to be used more easily
+will be the intrinsic object {{%ArrayPrototype%}} rather than
+{{%ObjectPrototype%}}.  This allows
+{{Array}} methods to be used more easily
```

Comment:
I *thought* we had agreed on not linking ES standard lib objects. I think we should be consistent here. And if we change our mind, link them all up in a separate PR.

Reply:
Well, you were actually more precise than that in https://github.com/heycam/webidl/pull/405#issue-249858976. But we're not linking Array anywhere else in the spec, are we?

Reply:
Oh heh, yeah. No, we are not currently linking to Array as an interface, but we did have a link to "Array methods" which I'm trying to eliminate in favor of something more generic, like this.

Would you want me to change this to `{{ECMAScript/Array}}` then?

Reply:
I don't know. Should we strive for consistency and link Array as we do Map and Set? (Note I opened #452 to discuss this further if needed it and cement it in the spec.)

Reply:
Made it consistent with Map/Set. Also fixed #452.